app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+  end
```

Comment:
Because we don't have an `:id` in the path in the routes.rb file, `params[:id]` will actually evaluate to `nil` and you cannot `find(nil)`.